format/leveldb/leveldb_log.go

```diff
@@ -0,0 +1,152 @@
+package leveldb
+
+// https://github.com/google/leveldb/blob/main/doc/log_format.md
+//
+// Files in LevelDB using this format include:
+//  - *.log
+//  - MANIFEST-*
+
+import (
+	"embed"
+
+	"github.com/wader/fq/format"
+	"github.com/wader/fq/internal/mathex"
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/interp"
+	"github.com/wader/fq/pkg/scalar"
+)
+
+//go:embed leveldb_log.md
+var leveldbLogFS embed.FS
+
+func init() {
+	interp.RegisterFormat(
+		format.LOG,
+		&decode.Format{
+			Description: "LevelDB Log",
+			Groups:      []*decode.Group{format.Probe},
```

Comment:
I was not quite aware what that `format.Probe` was doing (I copy pasted that part); now I am. Since the log files don't have magic strings of any kind in them, I think it's better to exclude them for now + from descriptors which is based on similar logic.

Reply:
👍 yeap there is no special probe code in fq, just that some decoders are tried if none is specified

Reply:
extra trivia: all formats and group exist as jq functions so you can do this ex: `... | probe`